### PR TITLE
Task misc nits

### PR DESCRIPTION
**Pull request by merveenoyan**
Misc nits: 
- Added lacking punctuations
- Added new resources to QA, text generation
- Fixed predictions on audio classification schema
- Fixed link in language identification
- Fixed use case consistency in audio classification
- Added Mean IoU metric to image segmentation 
and more :')

### CODE REVIEW

File: tasks/src/question-answering/about.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Would you like to learn more about QA? Awesome! Here you can find some curated r
 - [Question Answering Workshop](https://www.youtube.com/watch?v=Ihgk8kGLpIE&ab_channel=HuggingFace)
 - [How to Build an Open-Domain Question Answering System?](https://lilianweng.github.io/lil-log/2020/10/29/open-domain-question-answering.html) 
 - [Blog Post: ELI5 A Model for Open Domain Long Form Question Answering](https://yjernite.github.io/lfqa.html)
+- [Hugging Face Course Workshops: Question Answering](https://www.youtube.com/watch?v=Ihgk8kGLpIE&ab_channel=HuggingFace)
```
Comment (osanseviero) [inline tasks/src/question-answering/about.md:37]: This one is already there some lines above

File: tasks/src/sentence-similarity/about.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 You can extract information from documents using sentence similarity models. The first step is to rank documents using Passage Ranking models. You can then get the top-ranked document and search in it with sentence similarity models by picking the sentence that has the highest similarity to the input query.
 
 ## Sentence Transformers
-The [Sentence Transformers](https://www.sbert.net/) library is very powerful to compute embeddings of sentences, paragraphs and whole documents. An embedding is just a vector representation of a text, which makes embeddings very useful to find how similar two texts are. 
+The [Sentence Transformers](https://www.sbert.net/) library is very powerful to compute embeddings of sentences, paragraphs and whole documents. An embedding is just a vector representation of a text, and embeddings are useful to find how similar two texts are.
```
Comment (osanseviero) [inline tasks/src/sentence-similarity/about.md:6]:
```suggestion
The [Sentence Transformers](https://www.sbert.net/) library is very powerful to compute embeddings of sentences, paragraphs and whole documents. An embedding is just a vector representation of a text and are useful to find how similar two texts are. 
```

**Review by osanseviero — APPROVED** [review]
Thanks a lot for the changes!

File: tasks/src/question-answering/about.md (hunk 1 above)
Comment (merveenoyan) [reply tasks/src/question-answering/about.md]: Removed!
